### ISSUE TITLE
Add error messages for input length restrictions

### ISSUE THREAD

**Issue by svetlanaangelova** [samheather/iapt2group1#108]

**Comment by svetlanaangelova** [comment]
and improve error messages in general. No "Cannot be empty" or "Invalid username". Make them more specific.
